cli/cli/commands/cloud/load/load.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel/flags"
 	"github.com/kurtosis-tech/kurtosis/cli/cli/command_str_consts"
 	"github.com/kurtosis-tech/kurtosis/cli/cli/commands/kurtosis_context/add"
-	"github.com/kurtosis-tech/kurtosis/cli/cli/commands/kurtosis_context/context_switch"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/commands/kurtosis_context/set"
```

Comment:
I've renamed this package too to mirror the other commands

Reply:
thank you!!